13LessonThirteen/homework/index.html

```diff
@@ -1,10 +1,43 @@
 <!DOCTYPE html>
 <html>
+
 <head>
-  <title></title>
-  <link rel="stylesheet" href="css/style.css">
+    <meta charset="utf-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>Bone</title>
+    <link rel='stylesheet' id='google-work-sans-css' href='https://fonts.googleapis.com/css?family=Work+Sans:400,500,300' type='text/css' media='all' />
+    <link rel='stylesheet' id='google-roboto-mono-css' href='https://fonts.googleapis.com/css?family=Roboto+Mono:300,400,700' type='text/css' media='all' />
+    <link rel='stylesheet' id='google-open-sans-css' href='https://fonts.googleapis.com/css?family=Open+Sans%3A300italic%2C400italic%2C600italic%2C700italic%2C800italic%2C400%2C300%2C600%2C700%2C800&#038;subset=latin%2Ccyrillic-ext%2Cgreek-ext%2Cgreek%2Cvietnamese%2Clatin-ext%2Ccyrillic&#038;ver=4.4'
+        type='text/css' media='all' />
+    <link rel='stylesheet' id='google-lily-script-one-css' href='https://fonts.googleapis.com/css?family=Lily+Script+One&#038;subset=latin%2Clatin-ext&#038;ver=4.4' type='text/css' media='all' />
```

Comment:
FYI you can use google fonts to build a profile of all the fonts you want to include in your HTML, so it gives you a single `link` to embed in your code, rather than a separate link for each font. Remember, DRY (Don't Repeat Yourself) when possible.